packages/components/src/components/form/Switch/Switch.tsx

```diff
@@ -71,52 +70,80 @@ const CheckboxInput = styled.input`
     width: 1px;
 `;
 
+const Label = styled.div<Pick<SwitchProps, 'labelPosition' | 'isDisabled' | 'isAlert'>>`
+    display: flex;
+    justify-content: center;
+    margin-left: ${({ labelPosition }) => (labelPosition === 'before' ? '0px' : '16px')};
+    margin-right: ${({ labelPosition }) => (labelPosition === 'after' ? '0px' : '16px')};
+    color: ${({ isAlert, isDisabled, theme }) =>
+        getLabelColor(theme, { alert: isAlert, disabled: isDisabled })};
+    font-size: ${FONT_SIZE.NORMAL};
+    line-height: 24px;
+    text-align: left;
+`;
+
 export interface SwitchProps {
     id?: string;
     isChecked: boolean;
+    label?: string;
     onChange: (isChecked?: boolean) => void;
     isDisabled?: boolean;
-    isSmall?: boolean;
+    isAlert?: boolean;
     className?: string;
     dataTest?: string;
+    labelPosition?: 'before' | 'after';
 }
 
 export const Switch = ({
     onChange,
     id,
     isDisabled,
-    isSmall,
+    isAlert,
+    label,
     dataTest,
     isChecked,
     className,
+    labelPosition = 'after',
 }: SwitchProps) => {
     const handleChange = () => {
         if (isDisabled) return;
         onChange(!isChecked);
     };
 
     return (
-        <Container
-            isChecked={isChecked}
-            isDisabled={isDisabled}
-            isSmall={isSmall}
-            onClick={e => {
-                e.preventDefault();
-                handleChange();
-            }}
-            className={className}
-            data-test={dataTest}
-        >
-            <Handle isSmall={isSmall} isChecked={isChecked} disabled={isDisabled} type="button" />
```

Comment:
add `tabIndex={-1}` to the `Handle` so that only 1 element is focusable

Reply:
755f9fbbae98c6194cb5d379ff90e4ea918ce5f3